javascript.js

```diff
@@ -0,0 +1,22 @@
+const mobileMenu = document.getElementById('mobile-menu');
+const bars = document.getElementById('menu');
+bars.addEventListener('click', () => {
+  mobileMenu.style.display = 'flex';
+});
```

Comment:
* The mobile menu is not working kindly fix it 🙏

Reply:
- [ ] Mobile menu still not working. Looks like the path to your javascript file is broken and causes some of these errors. Please be sure you use the correct path. I left some code suggestions to fix this issue 
![Screenshot from 2021-07-04 02-23-52](https://user-images.githubusercontent.com/1642014/124368944-0b0fd680-dc6f-11eb-8d3e-2493ba6f32e9.png)

Reply:
Fixed